src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -237,6 +237,12 @@ public static void main(String[] args) throws Exception {
                 // Nothing to do here !
             }
 
+            try {
+              wrkld.setDeferConstraintChecks(xmlConfig.getBoolean("deferConstraintChecks"));
```

Comment:
Let's not use the term "deferred constraints" since that has a different meaning in postgresql and YSQL. Let's use `setForeignKeysAfterLoad` instead.

Reply:
Done.